ios/AppConfigSettingsFramework/ManagedAppConfigSettings.swift

```diff
@@ -50,10 +50,15 @@ open class ManagedAppConfigSettings: NSObject {
         }
         self.checkAppConfigChanges()
     }
-    
+
+    open func end()
+    {
+        NotificationCenter.default.removeObserver(self, name: UserDefaults.didChangeNotification, object: nil);
```

Comment:
Let's be consistent with the `start` method which calls `defaultCenter()`

Reply:
`start` method is calling `default` too. `defaultCenter()` is Swift 2.2 syntax.

Reply:
Oh cool, I had things opened locally - must had not ran the migration on my end 👍